src/queries/get-event-by-id.ts

```diff
@@ -0,0 +1,21 @@
+export const GET_EVENT_BY_ID = `
+  query ($eventId: ID) {
+    event(id: $eventId) {
```

Comment:
Probably you're missing `gql` before the query declaration?

```suggestion
  import {  gql } from "graphql-request"
 
    export const GET_EVENT_BY_ID = gql`
  query getEventById ($eventId: ID) {
    event(id: $eventId) {
      id
      title
      description
      eventUrl
      dateTime
      venue {
        id
        name
        address
        lat
        lng
      }
      image {
        baseUrl
      }
    }
  }
`;
```

Reply:
Actually that causes some eslint errors and I concluded that it's not worth working on it as `gql` just return string in the end.

Reply:
> Actually that causes some eslint errors and I concluded that it's not worth working on it as `gql` just return string in the end.

@Lada496 I see! That makes sense.